src/eip4824.ts

```diff
@@ -0,0 +1,155 @@
+    activityLogURI: `${baseUrl}/activities`,
+    governanceURI: space.website,
+    contractsURI: `${baseUrl}/contracts`
+  });
+});
+
+router.get('/:space/members', async (req, res) => {
+  let space: any = {};
+
+  try {
+    space = await getSpace(req.params.space);
+
+    if (!space.verified) return res.status(404).json({ error: 'INVALID' });
+  } catch (e) {
+    return res.status(404).json({ error: 'NOT_FOUND' });
+  }
+
+  const members = [...space.admins, ...space.moderators, ...space.members].map(
+    member => ({
+      type: 'EthereumAddress',
+      id: member
+    })
+  );
+
+  return res.json({
+    '@context': 'http://www.daostar.org/schemas',
+    type: 'DAO',
+    name: space.name,
+    members
+  });
+});
+
+router.get('/:space/proposals', async (req, res) => {
+  const id = req.params.space;
+  let space: any = {};
+  let proposals: any[] = [];
+
+  try {
+    space = await getSpace(id);
+
+    if (!space.verified) return res.status(404).json({ error: 'INVALID' });
+
+    proposals = await db.queryAsync(
+      'SELECT id, title, start, end FROM proposals WHERE space = ? ORDER BY created DESC LIMIT 20',
+      [id]
+    );
+  } catch (e) {
+    return res.status(404).json({ error: 'NOT_FOUND' });
+  }
+
+  const ts = Math.floor(Date.now() / 1000);
+  proposals = proposals.map(proposal => ({
+    type: 'proposal',
+    id: proposal.id,
+    name: proposal.title,
+    status:
+      ts > proposal.end ? 'closed' : ts > proposal.start ? 'active' : 'pending',
+    calls: []
+  }));
+
+  return res.json({
+    '@context': 'http://www.daostar.org/schemas',
+    type: 'DAO',
+    name: space.name,
+    proposals
+  });
+});
+
+router.get('/:space/activities', async (req, res) => {
+  const id = req.params.space;
+  let space: any = {};
+  let messages: any[] = [];
+
+  try {
+    space = await getSpace(req.params.space);
+
+    if (!space.verified) return res.status(404).json({ error: 'INVALID' });
+
+    messages = await sequencerDB.queryAsync(
+      'SELECT id, type, address FROM messages WHERE space = ? ORDER BY timestamp DESC LIMIT 20',
+      [id]
+    );
+  } catch (e) {
+    return res.status(404).json({ error: 'NOT_FOUND' });
+  }
+
+  const activities = messages.map(message => ({
+    id: message.id,
+    type: message.type,
+    member: {
+      type: 'EthereumAddress',
+      id: message.address
+    }
+  }));
+
+  return res.json({
+    '@context': 'http://www.daostar.org/schemas',
+    type: 'DAO',
+    name: space.name,
+    activities
+  });
+});
+
+router.get('/:space/contracts', async (req, res) => {
+  let space: any = {};
+
+  try {
+    space = await getSpace(req.params.space);
+
+    if (!space.verified) return res.status(404).json({ error: 'INVALID' });
+  } catch (e) {
+    return res.status(404).json({ error: 'NOT_FOUND' });
+  }
+
+  const contracts = space.treasuries.map(treasury => ({
+    type: 'EthereumAddress',
+    id: treasury.address,
+    name: treasury.name
+  }));
```

Comment:
<img width="779" alt="Untitled 13" src="https://github.com/snapshot-labs/snapshot-hub/assets/15967809/df72c6ad-7f2e-451e-992a-447d5c38c790">
`description` is optional? maybe we can return empty string for now?

Reply:
I think better not return anything, same for here: https://github.com/snapshot-labs/snapshot-hub/pull/797/commits/fafc3c7c83a28d00b87773370fcb5b9b67b8cf0f